spec/unit/ConfigurationTest.rb

```diff
@@ -21,28 +21,24 @@ def test_new_nonempty_string
   end
 
   def test_api_base
-    assert_equal 'http://api.local.dev:3000', Trolley::Configuration.new('key', 'secret', 'integration').apiBase
-    assert_equal 'https://api.trolley.com', Trolley::Configuration.new('key', 'secret', 'production').apiBase
-    assert_equal 'https://api.railz.io', Trolley::Configuration.new('key', 'secret', 'development').apiBase
-    assert_equal 'https://api.trolley.com', Trolley::Configuration.new('key', 'secret', 'non_standard_environment').apiBase
+    assert_equal 'https://api.trolley.com', Trolley::Configuration.new('key', 'secret').api_base
+    assert_equal 'https://api.railz.io', Trolley::Configuration.new('key', 'secret', api_base: 'https://api.railz.io').api_base
```

Comment:
Lets take out `*.railz.io` completely, and use something like `http://localhost:8080` or something instead